src/Queries/Select.php

```diff
@@ -8,6 +8,10 @@
 use SimpleCrud\Row;
 use SimpleCrud\Table;
 
+/**
+ * @method self where(string $condition, ...$bindInline)
+ * @method self limit(int $limit)
+ */
 class Select extends Query
```

Comment:
There are more methods (the list in the constant `ALLOWED_METHODS`). Not sure if it's a good idea because require more maintenance work.